lib/src/components/NxCheckbox/NxCheckbox.scss

```
@@ -4,13 +4,18 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+@import '../../scss-shared/_nx-focus-border-helpers';
```

Scss has goofy rules and shortcuts about its import paths. Generally the `_` and file extension aren't intended to be included, though it works fine if the are.
```suggestion
@import '../../scss-shared/nx-focus-border-helpers';
```

---

> Scss has goofy rules and shortcuts about its import paths. Generally the `_` and file extension aren't intended to be included, though it works fine if the are.

Fixed. 
c67fd99ba9304eb1436725fae06965fb906c349c